api/api.go

```diff
@@ -76,7 +77,7 @@ func request(method string, u url.URL, body io.Reader, headers ...http.Header) (
 
 // prettyTxIDError creates a support friendly error message with an transaction ID
 func prettyTxIDError(resp *http.Response) error {
-	if resp.Status == "429 Too Many Requests"  {
+	if resp.Status == strconv.Itoa(http.StatusTooManyRequests) + " " + http.StatusText(http.StatusTooManyRequests)  {
```

Comment:
Why does the string need to match? Why not just `resp.StatusCode` integer?? HTTP servers are free to put any arbitrary text after the integer. A `404 Not Found` response line and a `404 Where's it at?` response line is equivalent AFAIK.